docs/src/appendix/cheatcodes/start_mock_call.md

```diff
@@ -0,0 +1,55 @@
+# `start_mock_call`
+
+> `fn start_mock_call<T, impl TSerde: serde::Serde<T>, impl TDestruct: Destruct<T>>(contract_address: ContractAddress, function_name: felt252, ret_data: T)`
+
+Mocks contract call to a `function_name` of a contract at the given address. A call to function `function_name` will return data provided in `ret_data` argument. 
+
+If contract is deployed at given address, mocked function won't be executed. Address with no contract can be mocked as well.
+Mock can be canceled with [`stop_mock_call`](./stop_mock_call.md).
+
+- `contract_address` - target contract address
+- `function_name` - name of the function in a contract at the `contract_address` that will be mocked
+- `ret_data` - data to return by the function `function_name`
+
+> 📝 **Note**
+> The inner call (i.e. when a contract calls a function from within itself) cannot be mocked.
```

Comment:
```suggestion
> The inner call  - a call when a contract calls a function from within itself - cannot be mocked.
```

Reply:
I don't think this version is more readable. And the word `call` repeats too many times.

Reply:
Agree, can we remove the `i. e.` though? I think it is the only case of an inner call, no?

Reply:
It is used here to indicate clarification

Reply:
Thanks, mb, I was sure was en equivalent of giving an example!